### PR TITLE
Leverages ES6 in Notifications

### PR DESCRIPTION
**Pull request by VickyKoblinski**
This task was mostly to familiarize myself with a piece of Riot. 
* Most changes are to variable declarations
* Some functions changed to arrow functions
* Fixed typo

### CODE REVIEW

File: src/components/views/settings/Notifications.js [hunk 1]
```diff
@@ -15,33 +15,33 @@ limitations under the License.
 */
 
 'use strict';
-var React = require('react');
+const React = require('react');
```
Comment (ara4n) [inline src/components/views/settings/Notifications.js]: when updating to ES6, the correct way to fix these imports is actually to turn them into:
```
import React from 'react';
```
This technically works, but is very much not best practice.

**Comment by ara4n** [comment]
this is great - thanks! the only sticking point is to change the imports/requires to ES6-style for consistency with the rest of the update, but otherwise looks good to me.  (Interestingly, it looks like we have a blind spot when doing `()=>{ return foo }` rather than `()=>foo`, so thanks for catching these!)

**Comment by VickyKoblinski** [comment]
Sounds good to me! Changes have been made. ES6 is implicitly in strict mode, so I also removed `'use strict'`.

**Comment by ara4n** [comment]
awesome! lgtm, thanks again :)

**Comment by ara4n** [comment]
actually, there was one other catch: the PR is against master rather than develop, but I have changed the base in gihub successfully. technically you should also formally sign off the PR (see the DCO in contributing.rst) to confirm you are ok with the project licensing terms. going to merge anyway if you can put a DCO signoff in the comments here :)

**Comment by VickyKoblinski** [comment]
Whoops, I'm sorry about that! I had read to do that, then it slipped me. I look forward to contributing more to such a great project! :)

Signed-off-by: Vicky Koblinski vickykoblinski@gmail.com

**Pull request merged**